.travis.yml

```diff
@@ -13,6 +13,8 @@ matrix:
           env: TOXENV=py38
         - python: "3.9-dev"
           env: TOXENV=py39
+        - python: "3.8"
+          env: TOXENV=pre-commit
```

Comment:
Maybe this job should be allowed to fail.

Reply:
Allowing it to fail makes this completely pointless. Step back for a second and think about who the audience is, what we gain and what price we pay.

If you add this and allow failure then no one except for core developers will bother to look at the Travis results because Travis says "it's all OK". At the same time we lose because any added Travis job makes the tests run even longer. We gain nothing and lose out on loger Travis runs.

Reply:
Okay, you got me: We should either have it as normal job in Travis or not at all.

Reply:
I removed the Travis run and added a description for the `tox` environment which you can list with `tox -av` or `tox -lv`.